assets/dev/js/editor/controls/code.js

```diff
@@ -17,8 +17,9 @@ ControlCodeEditorItemView = ControlBaseDataView.extend( {
 			return;
 		}
 
-		var langTools = ace.require( 'ace/ext/language_tools' );
-
+		const langTools = ace.require( 'ace/ext/language_tools' ),
+			uiTheme = elementor.settings.editorPreferences.model.get( 'ui_theme' ),
+			userPrefersDark = window.matchMedia( '(prefers-color-scheme: dark)' ).matches;
```

Comment:
Omit the `window` and add an extra line after the consts declaration

Reply:
Done